bodhi/tests/server/test_models.py

```diff
@@ -128,6 +128,29 @@ def test_text_not_nullable(self):
         """
         self.assertEqual(model.Comment.__table__.columns['text'].nullable, False)
 
+    def test_get_unigue_testcase_feedback(self):
+        update = self.create_update(
+            (u'bodhi-2.3.3-1.fc24', u'python-fedora-atomic-composer-2016.3-1.fc24'))
+        package = update.builds[0].package
+        test1 = TestCase(name=u"Test 1", package=package)
+        test2 = TestCase(name=u"Test 2", package=package)
+        test3 = TestCase(name=u"Test 2", package=package)
```

Comment:
I wonder if we should put a uniqueness constraint on name and package so there won't be any duplicates? Of course, this would mean having to write a tricky DB migration too to remove the existing duplicates…